doc/arista_tp.md

```diff
@@ -0,0 +1,55 @@
+# Arista Traffic-Policy Use Notes
+
+# supported tokens
+
+The following tokens are supported:
+ - `action`
+ - `comment`
+ - `counter`
+ - `destination-address`
+ - `destination-exclude`
+ - `destination-port`
+ - `destination-prefix` - this should resolve to a configured field-set in traffic-policy format.
+ - `fragment-offset`
+ - `icmp-type`
+ - `logging`
+- `option`
+   - `established`
+   - `tcp-established`
+   - `initial`
+   - `rst`
+   - `first-fragment` - this  will be rendered as a `fragment` match.`
+ - `packet-length`
+ - `source-address`
+ - `source-exclude`
+ - `source-port`
```

Comment:
Can you add a section here that says # ports, and add the text -

In EOS traffic-policies, ports can be configured using:
source port <port-list> | field-set <port-set-name> ]
destination port <port-list> | field-set <port-set-name> ]
Currently, field-sets are not supported for ports, and only port lists are supported.